addons/common/XEH_postInit.sqf

```diff
@@ -153,6 +153,82 @@
     _unit enableIRLasers _mode;
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(forceFire), {
```

Comment:
Wait? Is this part of this PR?

Reply:
And this should be extracted into a function, if yes.

Reply:
I think I based this branch off other working stuff. I'll try to rebase it.

Reply:
476e6a6